04/app.js

```diff
@@ -4,6 +4,16 @@ const user = {
     born: {
         day: '14',
         month: '04',
-        year: '1985'
+        year: '1985',
+        birthday: function () {
+            birthday(this.day + "." + this.month);
+            return this.birthday
+        }
     }
-}
\ No newline at end of file
+}
+
+
+let today= new Date;
+
+console.log(today)
+console.log(user.born.birthday);
```

Comment:
Tutaj zwróć uwagę, że `today` ma swoje metody np. `todaj.getDate()` - to można wykorzystać :) 

PS. Analogicznie jest z miesiącem, tylko trzeba pamiętać, że są one liczone od 0